### PR TITLE
Upgrade dependencies and use Tuulbox logging

### PR DESCRIPTION
**Pull request by twyatt**
Snuck in fix to prevent crash on connection attempt failure and switch `jcenter` to `mavenCentral`.

### CODE REVIEW

**Comment by twyatt** [comment]
Blocked until Kable 0.6.0 released.

File: app/src/androidMain/kotlin/App.kt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+import android.app.Application
+import com.juul.tuulbox.logging.ConsoleLogger
+import com.juul.tuulbox.logging.Log
+
+class App : Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+        Log.dispatcher.install(ConsoleLogger)
```
Comment (cedrickcooke) [inline app/src/androidMain/kotlin/App.kt:11]: I'm now realizing I don't provide a Logcat logger in Tuulbox and this feels like a gap to me.

**Review by cedrickcooke — APPROVED** [review]

File: app/src/androidMain/kotlin/App.kt (hunk 1 above)
Comment (twyatt) [reply app/src/androidMain/kotlin/App.kt:11]: Agreed. Gonna be fun having Gradle properly handle the existence of a JVM **and** Android artifact. 😝

Comment (twyatt) [reply app/src/androidMain/kotlin/App.kt:11]: I _believe_ Coroutines did it by making the Android artifact separate from the "bundle" of multiplatform artifacts.

Screen Shot 2021-07-07 at 12 33 24 PM

Comment (twyatt) [reply app/src/androidMain/kotlin/App.kt:11]: https://github.com/JuulLabs/tuulbox/issues/74

**Pull request merged**